Tests/scripts/collect_tests_and_content_packs.py

```diff
@@ -1147,9 +1156,22 @@ def get_from_version_and_to_version_bounderies(all_modified_files_paths: set, id
         (string, string). The boundaries of the lowest from version (defaults to 0.0.0)
          and highest to version (defaults to 99.99.99)
     """
+    modified_metadata_list = modified_metadata_list if modified_metadata_list else set([])
     max_to_version = LooseVersion('0.0.0')
     min_from_version = LooseVersion('99.99.99')
     max_from_version = LooseVersion('0.0.0')
+
+    for metadata in modified_metadata_list:
+        pack_metadata_path = os.path.join(tools.pack_name_to_path(metadata), PACKS_PACK_META_FILE_NAME)
```

Comment:
Isn't `tools.pack_name_to_path(metadata)` return the path already?

Reply:
I'll update the variable names, it contains only the pack name.
I kept the original name - that was a mistake...